packages/example-phone/src/scripts/containers/page-sections/call-page-body.js

```diff
@@ -17,6 +17,7 @@ import {
 } from '../../actions/call';
 
 class CallPageBody extends Component {
+  /* eslint-disable react/no-unused-prop-types */
```

Comment:
Eslint doesn't recognize that the functions are actually called. I didn't want to modify the codebase of the example app too much since it is getting an overhaul soon. The errors are false positives, so I just ignored them.